roles/openshift_metrics/tasks/install_metrics.yaml

```diff
@@ -16,7 +16,11 @@
   include: install_heapster.yaml
   when: openshift_metrics_heapster_standalone | bool
 
-- find: paths={{ mktemp.stdout }}/templates patterns=*.yaml
+- name: Install Hawkular OpenShift Agent (HOSA)
+  include: install_hosa.yaml
+  when: openshift_metrics_install_hawkular_agent | bool
```

Comment:
i would update this to be `when: openshift_metrics_install_hawkular_agent | default(False) | bool` for safety